docs/usage.md

```diff
@@ -140,6 +151,7 @@ Metadata table (TSV) is the "Sample_preparations_sheet.tsv" that can be directly
 - **QBiC Code**: is needed to match metadata with the raw counts.
 - **Secondary Name**, samples will be named with the pattern: QBiC code + Secondary name.
 - **Condition: tag**: a separated column for each of the conditions. The headers of this columns start with "Condition: ". The values of these columns should not contain spaces.
+- **Data ID**: is needed to match metadata with the rsem or salmon files (not necessary for featurecounts).
```

Comment:
So what should be added to this Data ID column? If RNAseq is run with the QBIC IDs in the filename, will the Data ID be the QBiC ID?

Reply:
Ah, I forgot to remove that as well. The QBiC ID is sufficient.